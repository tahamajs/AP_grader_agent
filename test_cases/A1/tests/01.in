add_project ProjectA
add_task ProjectA Task1
add_task ProjectA Task2
list_projects
list_tasks ProjectA
