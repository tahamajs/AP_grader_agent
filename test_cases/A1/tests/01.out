Project added: ProjectA
Task added to ProjectA: Task1
Task added to ProjectA: Task2
Projects:
ProjectA
Tasks for ProjectA:
Task1
Task2
